addons/website/views/res_config_settings_views.xml

```diff
@@ -90,7 +90,7 @@
                             <div class="o_setting_right_pane" id="livechat_right_pane">
                                 <span class="o_form_label">Livechat</span>
                                 <div class="text-muted">
-                                    Alows your visitors to chat with you
+                                    Allows your visitors to chat with you
```

Comment:
@qsm-odoo Would you fix that in Odoo 16?
I'd not, but I am a bit too conservative maybe.
It means that once it's merged, all people around the world except the people using the software in English will have a term suddenly appearing in English and not understanding anything about it.
"Just" to fix an english typo.

Since it's now an "old" version, I don't know how fast are those new terms translated and imported back in Odoo.

Reply:
> Would you fix that in Odoo 16?

You know I would :wink: 

> It means that once it's merged [...]

This is indeed the Odoo rule: not fixing typo in stable.

Reply:
@qsm-odoo understood, I'll go ahead and close the PR for this version. However, would 17.2 still be considered stable if it's only been out for a couple weeks? Or should I only apply the change to master?

Reply:
You can definitely do it in 17.2, no user is on that version yet. You just have to update the .pot file alongside this change though (which would not be necessary for master).

Reply:
> I'll go ahead and close the PR for this version

And don't do that please, keep working on this PR, just change its target :+1:

Reply:
@rdeodoo @qsm-odoo target changed to 17.2, .pot added!